src/mapeo-project.js

```diff
@@ -627,10 +620,20 @@ export class MapeoProject extends TypedEmitter {
       this.preset.create(preset)
     )
     const createdPresets = await Promise.all(presetPromises)
-    const presetIds = createdPresets.map(({ docId }) => docId)
+    const sortedPresetIds = createdPresets
```

Comment:
I think it would be cleaner to return the presets already sorted but since *presets is a generator that returns a preset by one I haven't found a way to sort it there...
@gmaclennan wdyt?

Reply:
Yes I think easier / simpler to sort before we return from the preset reader. I added a comment with one way of doing it.